db/patterns/optable.eno

```diff
@@ -0,0 +1,12 @@
+name: Optable
+category: advertising
+website_url: https://www.optable.co/
+organization: optable
```

Comment:
```suggestion
organization: optable_technologies
```

Currently breaks tests, because optable is not known. Using the name from https://github.com/ghostery/trackerdb/pull/594/files should fix it.